save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/CreationView.kt

```diff
@@ -241,7 +241,8 @@ class CreationView : AbstractView<Props, ProjectSaveViewState>(true) {
             setState { isValidOrganization = true }
         }
 
-        if (fieldsMap[InputTypes.PROJECT_NAME].isNullOrBlank()) {
+        val projectName = fieldsMap[InputTypes.PROJECT_NAME]
+        if (projectName.isNullOrBlank() || projectName.contains(" ") || projectName.length > 64) {
```

Comment:
We have a common method for error message, but validation is duplicated in two places. Please extract validation into a method too

Reply:
Fixed